Misc/NEWS.d/next/Windows/2022-03-20-15-47-35.bpo-42658.16eXtb.rst

```diff
@@ -0,0 +1,3 @@
+Using Windows function ``LCMapStringEx`` in :func:`ntpath.normcase` instead
+of :func:`str.lower` to deal with non ASCII characters. The ``LCMapStringEx``
+is added in :mod:`winapi` module also.
```

Comment:
```suggestion
Support native Windows case-insensitive path comparisons by using
``LCMapStringEx`` instead of :func:`str.lower` in :func:`ntpath.normcase`.
Add ``LCMapStringEx`` to the :mod:`_winapi` module.
```